README.md

```diff
@@ -41,6 +41,19 @@ go build bamboo.go
 
 A deb package will be generated in `./builder` directory. You can copy to a server or publish to your own apt repository.
 
+Moreover, there is
+- a [Docker build container](builder/build.sh) which will generate the deb package in the volume mounted output directory.
+- and a [vagrant vm](Vagrantfile) where you could call docker build and docker run.
```

Comment:
Maybe specify why this is needed ? i.e a go binary built on a mac won't work on linux.